testing/image-loader-utils.ts

```diff
@@ -0,0 +1,37 @@
+import {act} from "@testing-library/react";
+
+/**
+ * Mocks image loading done by the ImageLoader component to immediately trigger
+ * the load event.
+ * @returns A function to unmock the image loading
+ */
+export const mockImageLoading = () => {
```

Comment:
Cool. I know we have at least one other place we do this in https://github.com/Khan/perseus/blob/e11c3f8071d2b0946351ff700c846c6c48fbebfa/packages/perseus/src/__tests__/renderer.test.tsx#L290

Reply:
Not asking for changes.  But could you leave a TODO comment there for future so if we touch that code we can refactor it to use this mock function instead?